src/NLog/Config/Factory.cs

```diff
@@ -246,7 +251,7 @@ public virtual TBaseType CreateInstance(string itemName)
                 message += ". Extension NLog.Web not included?";
 #endif
             }
-            else if (itemName?.StartsWith("database", StringComparison.OrdinalIgnoreCase)==true)
+            else if (itemName?.StartsWith("database", StringComparison.OrdinalIgnoreCase) == true)
             {
                 //common mistake and probably missing NLog.Web
```

Comment:
See some incorrect, copy-paste-comments. Maybe remove those?

Reply:
Maybe on a new PR? This hasn't changed here?

Reply:
all aren't that needed anymore. So removed them :)